src/invisible/explainers/enrichments/reference-image-mapper.md

```diff
@@ -0,0 +1,211 @@
+---
+description: Powerful tool to map gaze data from the scene camera onto a reference image 
+permalink: /invisible/explainers/enrichments/reference-image-mapper
+---
+
+# Reference Image Mapper
+
+Reference Image Mapper is a powerful tool to automatically map gaze onto to features/objects in the environment.
+
+<Youtube src="IF8f1Z3ZkEo" muted="1"/>
+
+
+A heatmap of gaze data mapped onto the reference image can be generated, and mapped gaze and fixation data can further be downloaded as [CSV files](/invisible/reference/export-formats.html#reference-image-mapper). 
+
+## Setup
+
+<Youtube src="ygqzQEzUIS4"/>
+
+As described in the setup video, you will need two things in addition to your eye tracking recording(s) to produce a Reference Image Mapper enrichment:
+1. A reference image
+2. A scanning video of the object/feature(s) taken with Pupil Invisible’s scene camera
+
+In this guide, we will show you how the Reference Image Mapper can be used in a few different situations. This should help you get set up using the enrichment in your own testing environment!
+
+Below, gaze is mapped in four very different environments: to a **magazine cover**, a **basketball backboard**, a **supermarket shelf**, and even a **whole building**!
+
+<PhotoGrid :images="[
+  require(`../../../media/invisible/rim/heatmap-0.jpg`),
+  require(`../../../media/invisible/rim/heatmap-1.jpg`),
+  require(`../../../media/invisible/rim/heatmap-2.jpg`),
+  require(`../../../media/invisible/rim/heatmap-3.jpg`),
+]"/>
+
+Let's take a look at what the Reference Image and Scanning Recording look like to produce these heatmaps.
+### 1. Magazine reading
```

Comment:
This overview is very functional, but the spacing between image, video and text could be nicer. I think fine for now, but we should touch this again during the Neon docs update!

Reply:
Agree. Perhaps adding an additional row below and tuning the margins and paddings on the captions